### PR TITLE
Feature/opt for multivalue datapoint

### PR DESCRIPTION
**Pull request by kwarunek**
Added multiValueDatapoint option to allow control of datapoint update strategy - append (current) vs. overwrite (like old graphite).

We have cases when we want to replace value. Of course it could be implemented on read, but since the old data is redundant, I think this change is cleaner and simpler.

### CODE REVIEW

**Comment by EinsamHauer** [comment]
@kwarunek,
I will check this out. 

But just to anticipate and explain why this was done in the first place - the problem is mostly in aggregation part. Basically, so far there is a good chance that several metrics being aggregated into one end up on one disthene server and several end up on another. As C\* doesn't provide (and I perfectly understand why) "SET x = x + 5" capability this is solved by using these weird lists.

Moreover even with a signle disthene aggregating it may turn out that some metrics being aggregated can come with a significant delay after the main batch was aggregated and flushed to C*. 

In other words this thing breaks the concept of aggregation right away unfortunately.

**Comment by kwarunek** [comment]
I agree. As I wrote maybe it is better to implement feature in disthene-reader that will enable to change the operation on data list... maybe

My case is that I would like to replace datapoints like in carbon-cache. And yes enabling this or any other similar feature probably would break aggregation, that's why it should be opt in/configurable.. Moreover like in our setup, there're external aggregation solutions (eg. carbon-c-relay it's pretty nice, grok+graylog+graphite-plugin, ...), so disthene receives already aggregated data.

**Comment by kwarunek** [comment]
I will rethink this 'cause as I'm digging I think I got your point, it will break aggregation in rollups that received later (after rollupsservice flush)... I fully understand that. For now I close the issue, but still we don't use this feature

btw: as I understand rollupservice calculate average (commonly already mean values) flush it and reader average those (if there are more than one) averages. Avg of avg of avg... I will back with issue after I take more thoughts
